test/makeRule.js

```diff
@@ -30,6 +30,11 @@ const parserOptions = {
         parserOptions,
         code: 'const x = gql`{ number }`',
       },
+      {
+        options,
+        parserOptions,
+        code: 'const x = segmented.TagName`height: 12px;`'
```

Comment:
A good test that @haikyuu wrote in #24 that might be useful to add here is to check that the tag `test.gql` doesn't get confused with `gql` and is correctly ignored.